templates/repo/clone_script.tmpl

```diff
@@ -14,8 +14,12 @@
 		if (httpsBtn) {
 			httpsBtn.textContent = window.origin.split(':')[0].toUpperCase();
 			httpsBtn.classList[!isSSH ? 'add' : 'remove']('primary');
+			httpsBtn.classList[!isSSH ? 'remove' : 'add']('basic');
+		}
+		if (sshBtn) {
+			sshBtn.classList[isSSH ? 'add' : 'remove']('primary');
+			sshBtn.classList[isSSH ? 'remove' : 'add']('basic');
```

Comment:
toggle?

Reply:
Can do, but no strong preference because if those buttons ever get into inconsistent state, these explicit remove/add would be more robust.

Reply:
>  because if those buttons ever get into inconsistent state, these explicit remove/add would be more robust.

Sorry I didn't the point.  IMO the `toggle` is robust enough and clearer.

```js
sshBtn.classList.toggle('primary', isSSH)
sshBtn.classList.toggle('basic', !isSSH)
```

Reply:
The point is if the first statement executes, but the second doesn't, the button will be in inconsistent state. But I guess practically this will never happen, so we can do the toggle for shorter and more readable code.

Reply:
> The point is if the first `classList` statement executes, but the second doesn't

Why the second doesn't execute?

https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/toggle

`toggle(token, force): force: If included, turns the toggle into a one way-only operation. If set to false, then token will only be removed, but not added. If set to true, then token will only be added, but not removed.`

Reply:
Ah yes, with that force flag it will behave the same.